systemtests/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.streamshub</groupId>
+        <artifactId>console</artifactId>
+        <version>0.7.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>systemtests</artifactId>
+    <description>
+        The intention of this module is to test StreamsHub Console UI with kafka instance deployed and managed by Strimzi Cluster Operator, to simulate a real user scenarios for end-to-end testing.
+    </description>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
These can probably be removed. The parent uses `maven.compiler.release`, which is a newer/preferred way to set the Java version and the source encoding has the same value in the parent.